contracts/terraswap_pair/src/contract.rs

```diff
@@ -757,7 +763,7 @@ pub fn assert_max_spread(
         let belief_price: Decimal256 = belief_price.into();
         let max_spread: Decimal256 = max_spread.into();
 
-        let expected_return = offer_amount / belief_price;
+        let expected_return = offer_amount.div_floor(belief_price);
```

Comment:
If used cosmwasm 1.0, dosen't support `div_floor`. 
We can chagned 
```let expected_return = offer_amount * (Decimal256::one() / belief_price);```

Reply:
https://github.com/terraswap/classic-terraswap/pull/14/commits/355c95881f7f3df7fe8a2915273e5dd8a8cf15a8